src/lib/EventListener/AdminExceptionListener.php

```diff
@@ -45,6 +47,8 @@ class AdminExceptionListener
     /** @var string */
     protected $kernelEnvironment;
 
+    use LoggerAwareTrait;
```

Comment:
Could you please move `use LoggerAwareTrait;` to the beginning of the class ?

Reply:
Done in 9927157fd14c028217cf764721d030303003f378